imports/server/migrations/35-reorganize-google-drive.ts

```diff
@@ -0,0 +1,22 @@
+import { Migrations } from 'meteor/percolate:migrations';
+import { _ } from 'meteor/underscore';
+import Documents from '../../lib/models/documents';
+import Hunts from '../../lib/models/hunts';
+import Puzzles from '../../lib/models/puzzles';
+import { ensureDocument, ensureHuntFolder } from '../gdrive';
+
+Migrations.add({
+  version: 35,
+  name: 'Migrate existing Google Drive files into new folder organization structure',
+  up() {
+    Hunts.find().forEach((hunt) => {
+      ensureHuntFolder(hunt);
```

Comment:
This approach seems fragile -- if Google integration is not configured but Hunts exist and have not been migrated, won't this throw when it reaches `checkClientOk();` and then the server won't come up, and you'll have to manually reset the migration lock.

Similarly, if the Google integration is disabled when this migration is applied, we'll never actually migrate the existing files into the different org structure.

Maybe this is better off either:

1. being reachable via an admin-only Meteor method that you call from the JS console in prod, and migrate manually, or
2. running on every startup with plenty of additional try-catch

I'm leaning towards 1 though?

Reply:
or

3. being reachable via a button on the setup page which also covers the "go do side effects until you've converged on the new directory layout under the new root folder" part of changing the setting?

Reply:
Yeah, OK, I think I'm convinced that this is a bad idea. I added a "reorganize google drive button" to the setup page, which I think should address both this and https://github.com/deathandmayhem/jolly-roger/pull/568#discussion_r783007491

Leaving this unrebased for the time being for ease of review